04_environment/index.tex

```diff
@@ -1 +1,286 @@
+\textbf{Disaster Total Damage} & 1000                & 200                 & 1200                & 1200                \\ \hline
+\textbf{CP's Threshold}        & 500                 & 500                 & 500                 & 500                 \\ \hline
+\textbf{CP's Current Value}    & 300                 & 300                 & 500                 & 700                 \\ \hline
+\textbf{CP's Multiplier}       & 1                   & 1                   & 0.5                 & 0.5                 \\ \hline
+\textbf{Mitigated Damage}      & 300                 & 200                 & 500                 & 1200                \\ \hline
+\textbf{Remaining Damage}      & 700                 & 0                   & 100                 & 0                   \\ \hline
+\textbf{CP's Remaining Value}  & 0                   & 100                 & 0                   & 100                 \\ \hline
+\end{tabular}
+\end{center}
+\caption{Demonstration of how the common pool works in different scenarios}
+\label{tab:Demonstration of how the common pool works in different scenarios}
+\end{table}
+
+In scenario 1, the disaster total damage is high, however there is no preparation from the islands through contribution to the common pool. As a result, the common pool could not mitigate all of the damage and 70\% of the damage will be directly on the islands.
+
+In scenario 2, even though there is no preparation, the common pool is still able to mitigate all of the damage thanks to the low damage of disaster. The leftover resource in the common pool will stay there.
+
+In scenario 3, the disaster damage is high. Fortunately, the islands have prepared themselves moderately through contribution to the common pool. Thus, the islands only have to take very little leftover damage.
+
+In scenario 4, the disaster damage is high, and the islands have prepared themselves very well. As a result, the damage is fully mitigated by the common pool, and there is still leftover resource in the pool. 
+
+
+\subsection{Future Work}
+
+To observe behaviors of agents in different settings, the future work of the common pool will be centered around varying the values of different parameters such as threshold and threshold multiplier. The forecasting function of the disaster part can be used as a tool to variate the common pool’s threshold and multiplier accordingly. In events that damage could be high, the preparation should be done more carefully, thus the common pool’s threshold can be raised to let islands know about potential damage from disaster. The same thing can be done for threshold multiplier, where the multiplier will be greater for bigger disaster. Since threshold and multiplier both encourage the preparation for disaster, one of them can be fixed at certain seasons to observe different behavior of agents when it comes to varying different parameters. 
```

Comment:
Since the future work might not happen I think generally people have been referring to it in the conditional tense i.e.

Future work on the common pool could be centred around ...

also american spelling of centred above
and should say future work *on* the common pool because future work happens on a subject not *of* it

Reply:
we are brits lol